lix/packages/sdk/src/open/openLix.ts

```diff
@@ -207,164 +266,190 @@ async function getChangeHistory({
 
 	const changes: any[] = [firstChange];
 
-	// TODO: walk change parents until depth
-	// await db
-	// 	.selectFrom("change")
-	// 	.select("id")
-	// 	.where((eb) => eb.ref("value", "->>").key("id"), "=", atomId)
-	// 	.where("type", "=", diffType)
-	// 	.where("file_id", "=", fileId)
-	// 	.where("plugin_key", "=", pluginKey)
-	// 	.where("commit_id", "is not", null)
-	// 	.executeTakeFirst()
-
 	return changes;
 }
 
 async function handleFileChange(args: {
+	queueEntry: any;
 	old: LixFile;
 	neu: LixFile;
 	plugins: LixPlugin[];
 	db: Kysely<LixDatabase>;
 }) {
 	const fileId = args.neu?.id ?? args.old?.id;
 
+	const pluginDiffs: any[] = [];
+
 	for (const plugin of args.plugins) {
-		const diffs = await plugin.diff?.file?.({
+		// glob expressions are expressed relative without leading / but path has leading /
+		if (!minimatch(normalizePath(args.neu.path), "/" + plugin.glob)) {
```

Comment:
@samuelstroschein  if its important it would be possible to use sqlite glob function directly with non table expressinons with the downside of being slower and i did not get this working with kysely, we can revisit this if it becomes a problem.